AWS.md

```diff
@@ -38,7 +38,20 @@ EC2
 
 VPC
 ---
-* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs.
+
+* Avoid using the default VPC - see [here](https://konekti.us/post/are-you-using-the-aws-default-vpc-stop-now/#:~:text=The%20default%20VPC%20lacks%20the,for%20auditing%20and%20troubleshooting%20purposes.) for a list of reasons why this is a bad idea. 
```

Comment:
We can remove the `#...` suffix from this I think. But I also think we should distill the things we care about here and inline it (a bit like Stack Overflow encourages in answers). This helps readers know what we (the Guardian) care about and saves them having to read that entire article - there is a lot of advice in it, some of which we don't (currently) follow. It also ensures our advice remains solid if that website disappears or the URL is taken down/modified.